README.md

```diff
@@ -1 +1,138 @@
-# Main branch
+# Mafia-District
+
+## Definitions
+
+- **Career** - refers to in-game profession which maybe assigns role-specific tasks
+- **Vampires** - is mafia role
+
+## **Service Boundaries**
+
+The following sections describe the key services and their responsibilities within the system.
+
+### **1. User Management Service**
+
+- **Responsibilities:**
+  - Manages user information
+  - Associates the user’s profile with device and location information
+  - Ensures that each player can create and manage a single profile
+- **Data Stored:**
+  - ID
+  - Email
+  - Username
+  - Password
+  - Device and location info
+
+---
+
+### **2. Game Service**
+
+- **Responsibilities:**
+  - Manages the primary game logic, including the Day/Night cycle and event notifications (kill/heal/rumor/visiting place)
+  - Tracks player status (alive/dead, role, career) in each game lobby (till 30 players)
+  - Initiates voting each night and tracks voting results. Also provides the list people that can be voted
+  - In-game currency
+- **Data Stored:**
+  - Player status (alive/dead)
+  - Player roles, career info and in-game currency
+
+---
+
+### **3. Shop Service**
+
+- **Responsibilities:**
+  - Allows players to buy in-game items using their currency
+  - Balance in item quantities
+  - Items available for purchase may include special items like garlic or water, used for defense or task completion
+- **Data Stored:**
+  - Item inventory
+  - Player purchases
+  - Transaction history
+
+---
+
+### **4. Roleplay Service**
+
+- **Responsibilities:**
+  - Manages the functionality of each in-game role and their specific abilities (Mafia ability to kill, Sheriff’s ability to guess roles)
+  - Tracks and records actions performed by players according to their roles
+  - Sends filtered announcements regarding player actions (e.g., murders, guesses) (probably send announcements to Game Service and Game Service to users)
+  - Handle role/ability for items
+- **Data Stored:**`
+  - Roles
+  - Roles actions
+  - Announcements related to actions
+
+---
+
+### **5. Town Service**
+
+- **Responsibilities:**
+  - Contains all locations within the town, including the Shop and Informator Bureau
+  - Tracks and records player movements within the town
+  - Reports player movements to the Task Service 1for task-related actions
+- **Data Stored:**
+  - Location data
+  - Player movement logs
+
+---
+
+### **6. Character Service**
+
+- **Responsibilities:**
+  - Allows players to customize their in-game characters
+  - Monitors the player’s inventory of purchased items from the Shop
+  - Manages customization options such as hair, clothing, and accessories
+- **Data Stored:**
+  - Customization options (e.g., hairstyle, coat)
+  - Inventory items purchased
```

Comment:
If customization is persistent, then the shop is not for game goodies, but for persistent customization, thus should be in the user service